### ISSUE TITLE
Displaying of PowerPoint presentations doesn't work

### ISSUE THREAD

**Issue by jlconlin** [MihajloNesic/jekyll-pdf-embed#9]
I tried to use jekyll-pdf-embed to display a presentation with: `{% pdf "presentation.pptx" %}`. This doesn't work and I get the following error:

<img width="447" alt="Screenshot 2023-05-02 at 11 05 38" src="https://user-images.githubusercontent.com/1275437/235735537-ab263164-4b6a-43f7-9010-499913f2c09f.png">

The "Learn more" link points to this: <https://r.office.microsoft.com/r/rlidowaviewhelp>.

Can someone provide a little more information about what is going on?

**Comment by MihajloNesic** [comment]
Where is your `presentation.pptx` file located? The path needs to be valid and pointing to the file (eg. /files/presentation.pptx)

You can view full examples [here](https://mihajlonesic.gitlab.io/jpe-examples/)

**Comment by jlconlin** [comment]
I verified that my presentation file was in the correct place. I have converted the PowerPoint to a PDF sitting right next to each other. `{% pdf presentation.pdf %}` works fine while `{% pdf presentation.pptx %}` does not. 

I don't know if the theme I'm using (Minimal Mistakes) is interfering or if it's because my files are inside a collection.

In my investigating, I tried giving the full path to the pdf/pptx. Because my files are inside of a collection, the beginning of the path begins with `_`. It wasn't obvious (perhaps it should have been) that---when giving the full path---it has to be the url path without the leading `_`.
